projects/billing/src/app/pages/billing/billing.component.html

```diff
@@ -43,7 +43,9 @@ <h3 class="fathym-header" *ngIf="SelectedPlan?.PlanGroup && SelectedPlan?.Interv
         </div>
         <div *ngIf="SelectedPlan?.PlanGroup === 'hobby'" class="important-note-container"fxLayout="row" fxFlex="70%">
           <span class="asterisk">*</span>
-          <div class="important-note">&nbsp;Fathym will only charge your credit card if you exceed the allocated API calls for the Hobby tier.</div>
+          <div *ngIf="State.LicenseType?.Lookup !== 'forecast'" class="important-note">&nbsp;Fathym will only charge your credit card if you exceed the allocated API calls for the Hobby tier.</div>
```

Comment:
These values should be coming from LCUServiceSettings, so they are configured as a part of the state when the app is setup, instead of casing by license type in here..  We should look anywhere else we might be keying on a specific license type and we need to pull that out... we can chat about this monday.